app/views/layouts/_admin_sidebar.html.haml

```diff
@@ -78,6 +78,9 @@
           - if can? :update, @conference.program.event_types.build
             %li{class: active_nav_li(admin_conference_program_event_types_path(@conference.short_title))}
               = link_to 'Event Types', admin_conference_program_event_types_path(@conference.short_title)
+          - if can? :update, @conference.program.event_types.build
```

Comment:
this is the same `if` as few lines up

Reply:
Sorry, this required a new entry in admin_ability.rb , I must have forgot to update it :sweat_smile:. Thanks! :smile: